tests/phpunit/tests/test-upgrade.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+class Upgrade_Test extends PLL_UnitTestCase {
+	protected $options_backup;
+
+	public static function wpSetUpBeforeClass( WP_UnitTest_Factory $factory ) {
+		parent::wpSetUpBeforeClass( $factory );
+
+		self::create_language( 'en_US' );
+		self::create_language( 'fr_FR' );
+	}
+
+	/**
+	 * @ticket #1664 see {https://github.com/polylang/polylang-pro/issues/1664}.
+	 */
+	public function test_upgrade_to_3_4() {
```

Comment:
I suggest to rename it maybe to `test_delete_transient_at_upgrade_to_3_4()` as here you don't really test the upgrade to 3.4. 
This could however be an idea to test the upgrade to 3.4 in 2 different tests (one for the strings translations, one for the fallback), maybe in a different PR though.

Reply:
Done in 0374d8fe